### PR TITLE
RC - D8 updates (LEB-1649, LEB-1634, LEB-1341, LEB-1452)

### PR DESCRIPTION
**Pull request by ynx**
LEB-1649 | The node's author name is no longer collected.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ install:
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
 
   # Install Drush.
-  - composer global require drush/drush
+  - composer global require drush/drush:8.*
```
Comment (ynx) [inline .travis.yml:28]: This is to fix test autorun on Travis - the default has been drush:9, which has stopped liking our tests.

File: src/Form/AdminSettingsForm.php [hunk 2]
```diff
@@ -78,7 +77,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['udf_touch_mappings'] = $this->buildUdfMappingsForm('touch');
     $form['udf_event_mappings'] = $this->buildUdfMappingsForm('event');
     $form['visibility'] = $this->buildVisibilityForm();
-    $form['advanced_configuration'] = $this->buildAdvancedConfigurationForm();
+    $form['advanced'] = $this->buildAdvancedForm();
```
Comment (ynx) [inline src/Form/AdminSettingsForm.php:80]: Rename "advanced_configuration" to "advanced". Function neutral.

File: src/Form/AdminSettingsForm.php [hunk 3]
```diff
@@ -239,13 +238,11 @@ private function buildFieldMappingsForm() {
...
-    $taxonomy_vocabulary_url_text = t('Taxonomy Vocabulary');
-    $taxonomy_vocabulary_url = Url::fromRoute('entity.taxonomy_vocabulary.collection', [], ['attributes' => ['target' => '_blank']]);
-    $taxonomy_vocabulary_link = Link::fromTextAndUrl($taxonomy_vocabulary_url_text, $taxonomy_vocabulary_url)->toString();
-
     $form = [
       '#title' => t('Field Mappings'),
-      '#description' => t('Create ') . $taxonomy_vocabulary_link . t(' and map to "content section", "content keywords", and "persona" fields.'),
+      '#description' => $this->t('Create <a href="@url" target="_blank">Taxonomy vocabularies</a> and map to "content section", "content keywords", and "persona" fields.', [
+        '@url' => Url::fromRoute('entity.taxonomy_vocabulary.collection')->toString(),
+      ]),
```
Comment (ynx) [inline src/Form/AdminSettingsForm.php:245]: Function neutral.

File: src/Service/Context/PageContext.php [hunk 4]
```diff
@@ -362,6 +360,7 @@ private function getJavaScriptTagRenderArray() {
         '#tag' => 'script',
         '#attributes' => [
           'src' => $this->assetsUrl . '/' . SELF::LIFT_JS_FILENAME,
+          'async' => true,
```
Comment (ynx) [inline src/Service/Context/PageContext.php:363]: Adds "aync" to `lift.js`'s script tag.

File: src/Tests/SettingsTest.php [hunk 5]
```diff
@@ -137,9 +140,13 @@ public function testMetatagsAndScriptTag() {
     $this->assertRaw('node page', '[testMetatagsAndScriptTag]: page_type metatag value is loaded on the node page.');
     $this->assertRaw('acquia_lift:account_id', '[testMetatagsAndScriptTag]: account_id metatag is loaded on the node page.');
     $this->assertRaw('AccountId1', '[testMetatagsAndScriptTag]: account_id metatag value is loaded on the node page.');
-    $this->assertRaw('acquia_lift:contentReplacementMode', '[testMetatagsAndScriptTag]: content replacement mode metatag value is loaded on the node page.');
+    $this->assertRaw('acquia_lift:contentReplacementMode', '[testMetatagsAndScriptTag]: content replacement mode metatag is loaded on the node page.');
+    $this->assertRaw('customized', '[testMetatagsAndScriptTag]: content replacement mode metatag value is loaded on the node page.');
+    $this->assertRaw('acquia_lift:contentOrigin', '[testMetatagsAndScriptTag]: content origin metatag is loaded on the node page.');
+    $this->assertRaw('08c93130-2e45-45f6-af6d-7c02de8cd90c', '[testMetatagsAndScriptTag]: content origin metatag value is loaded on the node page.');
```
Comment (ynx) [inline src/Tests/SettingsTest.php:146]: Tests.

File: src/Service/Context/PageContext.php [hunk 6]
```diff
@@ -232,7 +231,6 @@ private function setNodeData(NodeInterface $node) {
     $this->htmlHeadContexts['published_date'] = $node->getCreatedTime();
     $this->htmlHeadContexts['post_id'] = $node->id();
     $this->htmlHeadContexts['content_uuid'] = $node->uuid();
-    $this->htmlHeadContexts['author'] = $node->getOwner()->getUsername();
```
Comment (ynx) [inline src/Service/Context/PageContext.php:235]: No longer surface / collect author information.

**Review by danikalmar — APPROVED** [review]
looks good!

**Comment by ynx** [comment]
Thank you @danikalmar for the review!

**Pull request merged**